### ISSUE TITLE
flash feit bulbs 

### ISSUE THREAD

**Issue by steveysaurus** [steveysaurus/hassio-victoria#1]
https://github.com/ct-Open-Source/tuya-convert

will need another SD card or backup current install..

**Comment by steveysaurus** [comment]
Roundabout solution:
Took a snapshot of the Hassio instance. Saved it outside of the SD card in Rasberry Pi. Will install Rasbian Stretch 4.14 (>2gb!) to perform the Tuya-convert procedure.
Then plan to perform a new install of Hassio and reload the snapshot.
Remove bulbs from previous config and replace with updated bulbs.
Consider getting another SD card vs ditching Hassio and do a Rasbian build. The latter would allow a distinct MQTT broker among other things, but ultimately could make this hobby more intense.

**Comment by steveysaurus** [comment]
[Tuya-Convert](https://github.com/ct-Open-Source/tuya-convert) worked well one 2 of 4 bulbs. After much frustration, log reading, and web crawling, I believe that 2 were auto-updated by the Feit app to a firmware that Tuya-convert could not flash. I installed two (they came 2 per pack) bulbs and synced them to the default app to test if HomeAssistant, Alexa, HomeKit, etc could sync. Turns out only Alexa could, so I googled and discovered that the SmartLife app was the only app (at the time) that could discover and sync the bulbs with Hassio. So the next two bulbs were not synced with Feit but SmartLife only and these were the bulbs that I could flash new firmware onto easily. As a tangent I did receive a robocall in Chinese a day or two after registering my phone number with the SmartLife app--observation bias, or perhaps the concerns regarding security may be a real concern. Also the latency was bugging me, which has significantly improved since flashing to Tasmota. 

I'd recommend sticking with the default core version 2.3.0 Sonoff-Tasmota 6.5 that comes default. I updated one bulb to 2.5 full and the bulb was going crazy: cycling through four stages of LEDs over a 2 second cycle non-stop. It also appeared to be cycling through connecting and disconnecting from the WiFi with each cycle. Pings would time out, except for when they didn't. Flashing to Tasmota has also seemed to remove the reset and AP mode mechanism of turning the switch on/off 3 or 6 times. What seemed to stabilize the situation was connecting my work Windows PC to the network with device discovery mode on. I had tried things like disabling the password on the network, rebooting the router, and power cycling the device.. Immediately, it stopped for a couple of minutes before restarting the cycle of frustration. After either chance or reconnecting the Windows PC to the network, I quickly updated the firmware to an older version OTA (over-the-air), which didn't seem to really downgrade the firmware, but it did stabilize the condition. 

Recommend setting Wificonfig 2 (wifi goes to AP mode if locked-out); option17 1 (color to coordinates), and option37 54 (recalibrate the color coordinates to match the R,G,B,W,W schema). 

Set the template to: `{"NAME":"OM60/RGBW","GPIO":[255,255,255,255,140,37,0,0,38,142,141,255,255],"FLAG":0,"BASE":18}`
https://blakadder.github.io/templates/feit-a19.html
